### PR TITLE
DDF-2296: Fixed local resource query to include anytag search

### PR DESCRIPTION
**Pull request by rzwiefel**
#### What does this PR do?
Adds back the anytag search into the catalog framework queries while getting resources

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@djblue 
@ryeats 
@stustison 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@coyotesqrl
@pklinef

### CODE REVIEW

**Comment by stustison** [comment]
Sounds like a bug fix to me. 2.9.x?

File: catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java [hunk 1]
```diff
@@ -2734,6 +2734,34 @@ protected Query createMetacardIdQuery(String metacardId) {
...
+                .attribute(Metacard.TAGS)
+                .is()
+                .like()
+                .text(FilterDelegate.WILDCARD_CHAR);
+        Filter filter = frameworkProperties.getFilterBuilder()
+                .allOf(anyTag, query);
+        return new QueryImpl(filter);
+    }
+
+    protected Query createMetacardIdQueryWithAnyTags(String metacardId) {
```
Comment (coyotesqrl) [inline catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java]: Is this unused method here for future-proofing?

File: catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java [hunk 2]
```diff
@@ -2734,6 +2734,34 @@ protected Query createMetacardIdQuery(String metacardId) {
         return createPropertyIsEqualToQuery(Metacard.ID, metacardId);
     }
 
+    protected Query anyTag(Query query) {
```
Comment (coyotesqrl) [inline catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java]: No test method?

**Comment by rzwiefel** [comment]
Put up as a new PR since this is obsolete after CatalogFramework changes.

**Pull request closed without merging**